footer/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import "../index.css"
+
+const Footer = () => {
+  return (
+    <div className="footer">
+      <div className="footerInner">
+        <p>
+          Copyright © 2021 <a href="https://gmcruz.me/debugs-bunnies/">DebugsBunnies POD 2.0.3 </a>  |{' '}
```

Comment:
Instead of this we should put "Made with :heart: by Debugs Bunnies pod 2.0.3"

Reply:
Updating